build.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# Builds pair a scala version with an sbt version
+SCALA_VERSIONS=(  2.12.11   2.12.12)
+SBT_VERSIONS=(    1.4.7     1.4.6)
+
+# $((x - 1)) is needed because otherwise it actually outputs length, but we need to be zero-indexed
+for i in $(seq 0 $((${#SCALA_VERSIONS[@]} - 1))); do
+  IMAGE_NAME="us.gcr.io/$PROJECT/scala-baseimage:jdk11-${SCALA_VERSIONS[i]}-${SBT_VERSIONS[i]}"
```

Comment:
where is `$PROJECT` populated from? I see that the cloud build yaml specifies it; does that mean this build.sh can only be run successfully via cloud build?

Reply:
I'd just been feeding it in manually, I may remove it altogether once we know the actual publish coordinates, though.